src/content/en/updates/2017/04/devtools-release-notes.md

```diff
@@ -50,6 +48,24 @@ that you need.
   </figcaption>
 </figure>
 
+Clicking on a URL reveals that file in the **Sources** panel with a breakdown
+of which lines of code executed.
+
+<figure>
+  <img src="/web/updates/images/2017/04/coverage-breakdown.png"
+       alt="A breakdown of code coverage in the Sources panel"/>
+  <figcaption>
+    <b>Figure 2</b>. A breakdown of code coverage in the <b>Sources</b> panel
+  </figcaption>
+</figure>
+
+Each line of code is color-coded:
+
+* Solid green means that line of code executed.
+* Solid red means it did not execute.
+* Red and green means that only some code on that line executed. For example,
+  a ternary expression like `var b = (a > 0) ? a : 0` is colored like this.
```

Comment:
I intended for the previous sentence to set the context. So, a ternary expression is colored red and green.